src/main/java/com/ypm/service/youtube/VideoServiceImp.java

```diff
@@ -68,9 +72,32 @@ public void deleteVideo(String accessToken, String videoId) throws IOException {
     }
 
     @Override
-    public List<PlaylistItem> getPlayListVideos(String accessToken,
-                                                String playListId) throws IOException {
+    public List<VideoDto> getVideoData(List<String> videoIds) throws IOException {
+        final int maxResults = 50;
 
+        // Remove duplicates from the incoming list with ids
```

Comment:
I believe that the comment here is quite good, we can try to agree on some prefix like `DEVNOTE` just to clarify that the comment here is left for additional explanation of the logic)

Reply:
I think the idea with `DEVNOTE` is great, so I'll prefix the comment with it

Reply:
Though, let's make it `DEV-NOTE` for slightly better readability.

Also, if we want to make them stand out, we can add a pattern in IntelliJ IDEA so dev-notes are treated as TODOs

Pattern:
`\bdev-note\b.*`

<img width="1094" alt="Screenshot 2024-08-31 at 16 20 47" src="https://github.com/user-attachments/assets/e635b866-2810-412a-b238-2fba0e52174b">
<img width="403" alt="Screenshot 2024-08-31 at 16 20 58" src="https://github.com/user-attachments/assets/42ab1c89-3b61-4630-b9d5-2a7ac1f8cfe8">

Reply:
Agreed, thanks!!